app/services/food_service.rb

```diff
@@ -0,0 +1,12 @@
+class FoodService
+  def self.conn 
+    Faraday.new("https://api.nal.usda.gov/fdc/v1/foods") do |faraday|
```

Comment:
Typically the Faraday connection will just include the base URL, so everything after `.gov` can be moved to the path on line 9